src/app/[locale]/viewer/_components/AASOverviewCard.tsx

```diff
@@ -69,25 +70,30 @@ export function AASOverviewCard(props: AASOverviewCardProps) {
     async function createAndSetUrlForImageFile() {
         if (!props.aas) return;
 
-        try {
-            let image: Blob;
-            if (registryAasData) {
-                const registryRepository = AssetAdministrationShellRepositoryApi.create({
-                    basePath: registryAasData.aasRegistryRepositoryOrigin,
-                });
-                image = await registryRepository.getThumbnailFromShell(props.aas.id);
+        let image: Blob;
+        if (registryAasData) {
+            const registryRepository = AssetAdministrationShellRepositoryApi.create(
+                mnestixFetch(),
+                undefined,
+                registryAasData.aasRegistryRepositoryOrigin,
+            );
+            const result = await registryRepository.getThumbnailFromShell(props.aas.id);
```

Comment:
Again, I'd rename "result" to "response" to avoid confusion here.

Reply:
Good catch